docs/zh/UserGuide/Query-Data/Select-Expression.md

```diff
@@ -106,6 +110,55 @@ Total line number = 5
 It costs 0.014s
 ```
 
+## 逻辑运算查询
+#### 一元逻辑运算符
+支持运算符 `!`
+
+输入数据类型：`BOOLEAN`
+
+输出数据类型：`BOOLEAN`
+
+注意：`!`的优先级很高，记得使用括号调整优先级
+
+#### 二元比较运算符
+
+支持运算符 `>`, `>=`, `<`, `<=`, `==`, `!=`
+
+输入数据类型： `INT32`, `INT64`, `FLOAT`, `DOUBLE`
+
+会将所有数据转换为`DOUBLE`类型后进行比较。
+
+返回类型：`BOOLEAN`
+
+#### 二元逻辑运算符
+
+支持运算符 AND:`and`,`&`, `&&`; OR:`or`,`|`,`||`
+
+输入数据类型：`BOOLEAN`
+
+返回类型 `BOOLEAN`
+
+注意：当某个时间戳下左操作数和右操作数都为BOOLEAN类型时，二元逻辑操作才会有输出结果
```

Comment:
BOOLEAN -> `BOOLEAN`
It's better to add a summary to indicate the priority of all the operators. Now the `&` and `&&` are both the same with the meaning `boolean and`. Is it `&` for `bitwise and` and `&&` for `boolean and` better? What do you think? @SteveYurongSu @flashzxi

Reply:
It seems that the `&` and `&&` have been  already defined with the same meaning. There will be a break change if we modify it. :(
Just add a summary of the priorites of all the operators. Thank you~

Reply:
Got it. ok